src/plugins/console/server/lib/spec_definitions/js/query/dsl.ts

```diff
@@ -745,6 +745,28 @@ export const query = (specService: SpecDefinitionsService) => {
         // populated by a global rule
       },
     },
+    script_score: {
+      __template: {
+        query: '',
+        script: {},
+      },
```

Comment:
```suggestion
      },
      query: {},
```
`query` should be added outside `__template` too because if the user deletes the initial autocomplete object, they need to see `query` in the list of properties. Empty object because it's autocompleted by a global rule

Reply:
Fixed